libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/TabsView/TabsView.tsx

```diff
@@ -168,28 +183,61 @@ export class TabsView extends React.PureComponent<
               )}
               {t.key === GlobalTabKeys.FeatureImportancesTab &&
                 this.props.modelExplanationData?.[0] && (
-                  <FeatureImportancesTab
-                    modelMetadata={this.props.modelMetadata}
-                    modelExplanationData={this.props.modelExplanationData}
-                    selectedWeightVector={this.state.selectedWeightVector}
-                    weightVectorOptions={this.state.weightVectorOptions}
-                    weightVectorLabels={this.state.weightVectorLabels}
-                    requestPredictions={this.props.requestPredictions}
-                    onWeightVectorChange={this.onWeightVectorChange}
-                  />
+                  <>
+                    <div className={classNames.sectionHeader}>
+                      <Text variant={"xxLarge"} id="featureImportanceHeader">
+                        {
+                          localization.ModelAssessment.ComponentNames
+                            .FeatureImportances
+                        }
+                      </Text>
+                    </div>
+                    <FeatureImportancesTab
+                      modelMetadata={this.props.modelMetadata}
+                      modelExplanationData={this.props.modelExplanationData}
+                      selectedWeightVector={this.state.selectedWeightVector}
+                      weightVectorOptions={this.state.weightVectorOptions}
+                      weightVectorLabels={this.state.weightVectorLabels}
+                      requestPredictions={this.props.requestPredictions}
+                      onWeightVectorChange={this.onWeightVectorChange}
+                    />
+                  </>
                 )}
               {t.key === GlobalTabKeys.CausalAnalysisTab &&
                 this.props.causalAnalysisData?.[0] && (
-                  <CausalInsightsTab
-                    data={this.props.causalAnalysisData?.[0]}
-                  />
+                  <>
+                    <div
+                      className={classNames.sectionHeader}
+                      id="causalAnalysisHeader"
+                    >
+                      <Text variant={"xxLarge"} id="causalInsightsTab">
+                        {
+                          localization.ModelAssessment.ComponentNames
+                            .CausalAnalysis
+                        }
+                      </Text>
+                    </div>
+                    <CausalInsightsTab
+                      data={this.props.causalAnalysisData?.[0]}
+                    />
+                  </>
                 )}
 
               {t.key === GlobalTabKeys.CounterfactualsTab &&
                 this.props.counterfactualData?.[0] && (
-                  <CounterfactualsTab
-                    data={this.props.counterfactualData?.[0]}
-                  />
+                  <>
```

Comment:
I see this sectionHeader div section with the component name repeated several times.  I think it can be moved into its own component or function to simplify and reduce duplication.

Reply:
Now all the component names have been moved to TabsView to have a unified style